README.md

```diff
@@ -36,6 +36,8 @@ Please follow your platform's instructions on how to do that.
 1.  [git](http://git-scm.com) must be installed to download the source
     code for the DC/OS CLI.
 2.  [python](https://www.python.org/) version 3.7.x must be installed.
+    If you have another Python version installed, you can export
+    `DCOS_EXPERIMENTAL=1` and the installation steps will allow it.
```

Comment:
I'm not sure I understand what this means. What does `DCOS_EXPERIMENTAL=1` allow that I would otherwise not be able to do?

Reply:
I've rephrased it, let me know if that's more clear.